components/datatypes/auditing/auditable.schema.json

```diff
@@ -43,6 +43,12 @@
           "description": "The last dataset files in Catalog which has modified the record. At creation time, `modifiedByBatchID` is set as `createdByBatchID`.",
           "meta:titleId": "auditable##xdm:modifiedByBatchID##title##45371",
           "meta:descriptionId": "auditable##xdm:modifiedByBatchID##description##21001"
+        },
```

Comment:
@anandphatak - I noticed this auditable data type already exists but is not used in very many places. Would it be best to update this or should we create a new one with this new property?

Reply:
This datatype is actually used in many places. I am curious, why would you want to include `expirationDate` as a part of audit log? Audit fields have a different purpose, which is to track who created or modified a record. Expiration date field gives me an impression that you want to add some logic to delete the record..

Reply:
Sounds like this does not belong here. I will put this in a different datatype as we would like to use some different names for some of the other audit properties as well.